web/src/config.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
You shouldn't do unnecessary changes :) And it would be good idea to clean a history a little bit after you'll finish with changes. I'll show you how.)